static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -124,6 +136,66 @@ export function getDefaulServerlessImports({
   return imports;
 }
 
+export function getNodeRunCommandSnippet(): string {
+  return `# If you are using CommonJS (CJS)
+node --require ./instrument.js app.js
+
+# If you are using ECMAScript Modules (ESM) - Note that this is only available from Node 18.19.0 onwards.
+node --import ./instrument.mjs app.mjs`;
+}
+
+export function getImportInstrumentSnippet(): string {
+  return `// IMPORTANT: Make sure to import \`instrument.js'\` at the top of your file.
+  // If you're using ECMAScript Modules (ESM), use \`import './instrument.js';\`
+  require('./instrument.js');`;
+}
+
+export const getSdkInitSnippet = (
+  params: DocsParams,
+  sdkImport: 'node' | 'aws' | 'gpc'
+) =>
+  `${
+    sdkImport === 'node'
+      ? getDefaultNodeImports({productSelection: getProductSelectionMap(params)}).join(
+          '\n'
+        )
+      : sdkImport === 'aws'
+        ? getDefaultServerlessImports({
+            productSelection: getProductSelectionMap(params),
+            library: 'aws-serverless',
+          }).join('\n')
+        : sdkImport === 'gpc'
+          ? getDefaultServerlessImports({
+              productSelection: getProductSelectionMap(params),
+              library: 'google-cloud-serverless',
+            }).join('\n')
+          : ''
+  }
+
+Sentry.init({
+  dsn: "${params.dsn}",
+  ${
+    params.isProfilingSelected
+      ? `integrations: [
+    nodeProfilingIntegration(),
+  ],`
+      : ''
+  }${
+    params.isPerformanceSelected
+      ? `
+      // Performance Monitoring
+      tracesSampleRate: 1.0, //  Capture 100% of the transactions`
```

Comment:
l/feel free to disregard for now: I think we should remove all references to "transactions" in SDK-specific documentation:

```suggestion
      tracesSampleRate: 1.0, //  Capture 100% of spans`
```

Now that being said, I'm realizing that capturing 100% of spans is also misleading because the sample rate is only applied to root/segment spans 😢 

It's not too important though and we should also iterate to get this first set of changes in today. So we can just follow up on this later :)

(leaving this comment here for context)